client/src/components/Common/MiddleTruncatedText/index.jsx

```diff
@@ -3,13 +3,14 @@ import { Tooltip, OverlayTrigger } from 'react-bootstrap';
 import { getEndString } from '../../../helpers/format';
 import './MiddleTruncatedText.scss';
 
-export const MiddleTruncatedText = ({ children, end, placement = 'top' }) => {
+export const MiddleTruncatedText = ({ children, end, start, placement = 'top' }) => {
```

Comment:
this should be exported as default

Reply:
Hi @duc-q , this component used in other components, so I will create another PR for it :D